services/movie_session.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from db.models import MovieSession, Movie, CinemaHall
+from django.db.models import QuerySet
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    query = MovieSession.objects.all()
+
+    if session_date:
+        query = MovieSession.objects.filter(show_time__date=session_date)
+
+    return query
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if movie_id:
+        movie = Movie.objects.get(id=movie_id)
```

Comment:
You dont really need import Movie class and use it to change movie_id for MovieSession. You have movie_session instance and you can set id by next way:
movie_session.movie_id = movie_id